ui/src/app/job-details/tasks/tasks.component.css

```diff
@@ -52,3 +52,24 @@
   cursor: pointer;
   text-decoration: none;
 }
+
+clr-tooltip-content {
+  background-color: rgba(0, 0, 0, 0.65);
+  padding: 0.25rem 0.5rem;
+  font-size: 0.65rem;
+  color: #fff;
+  display: block;
+  margin: -0.6rem 0.5rem 0 0;
+}
+
+clr-icon {
+  margin-right: 1rem;
+}
+
+::ng-deep clr-icon.has-alert.log-error svg.can-alert path.clr-i-alert.clr-i-outline--alerted {
```

Comment:
`::ng-deep` I think should be avoided if possible. This CSS selector is also pretty complicated and I could imagine it breaking if we upgraded our clarity icons version, for example. Have you looked into whether that is any other way to apply this style?

Reply:
I wasn't able to get this to work without using ::ng-deep, but I'm definitely open to not using it.  I can definitely shorten the list of selectors.